Starbucks/Controllers/CartViewController/CartViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  CartViewController.swift
+//  Starbucks
+//
+//  Created by Reuben Simphiwe Kuse on 2024/04/25.
+//
+
+import Foundation
+import UIKit
+
+class CartViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    
+    // Dummy array of products in the cart
+    var cartProducts: [Product] = []
+    
+    
+    lazy var cartTableView: UITableView = {
+        let tableView = UITableView(frame: .zero, style: .plain)
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(ProductTableViewCell.self, forCellReuseIdentifier: "ProductTableViewCellID")
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        return tableView
+    }()
+    
+    lazy var emptyCartLabel: UILabel = {
+        let label = UILabel()
+        label.text = "You have no items in your cart. Start shopping."
+        label.textColor = StarbucksColors.starbucksTextGray
+        label.textAlignment = .center
+        label.numberOfLines = 2
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = .cyan
+        setupUI()
+        
+    }
+    
+    func setupUI() {
+        
+        view.addSubview(cartTableView)
+        view.addSubview(emptyCartLabel)
+        
+        emptyCartLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        emptyCartLabel.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+        
+        cartTableView.topAnchor.constraint(equalTo: view.topAnchor).isActive = true
+        cartTableView.leftAnchor.constraint(equalTo: view.leftAnchor).isActive = true
+        cartTableView.rightAnchor.constraint(equalTo: view.leftAnchor).isActive = true
+        cartTableView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
+        
+    }
+    
+    func registerCell() {
+        cartTableView.register(ProductTableViewCell.self, forCellReuseIdentifier: "ProductTableViewCellID")
+           
+    }
```

Comment:
You are registering your cell twice. You already registered it in `lazy var cartTableView: UITableView...`